plugins/aws/importers.go

```diff
@@ -94,10 +97,27 @@ var backendNames = map[keyring.BackendType]string{
 	keyring.PassBackend:          "Pass",
 }
 
+func fileKeyringPassphrasePrompt(prompt string) (string, error) {
+	if password, ok := os.LookupEnv("AWS_VAULT_FILE_PASSPHRASE"); ok {
+		return password, nil
+	}
+
+	fmt.Fprintf(os.Stderr, "%s: ", prompt)
```

Comment:
This doesn't show up when running a local profile through RPC. However, the line underneath for the `Stdin` works as expected. @jpcoenen do you know why that is, and how could we overcome that? This is the only aspect that blocks this PR.

Reply:
Why are we printing to Stderr?

Reply:
Apologies; I missed this comment originally. I think there is a way to read the output a plugin writes to `stderr`, but I am not sure whether that is the preferred way to go here. 

Might be interesting to see if there are any Terraform Providers that prompt the users. If so, we could see how they tackle that.